.circleci/config.yml

```diff
@@ -8,6 +8,7 @@ jobs:
 
         steps:
             - checkout
+            - run: sudo apt-get install portaudio19-dev python3-pyaudio
```

Comment:
you're using a virtualenv and installing pyaudio with pip, do you need to install python3-pyaudio?